packages/gallery/tests/drivers/mocks/items.js

```diff
@@ -422,6 +422,80 @@ const images = [
   },
 ];
 
+const itemsWithDirectShareLink = [
+  {
+    directShareLink:
+      'https://static.wixstatic.com/media/8bb438_1b73a6b067b24175bd087e86613bd00c.jpg',
```

Comment:
Looks like all these objects are duplicated, except for one parameter (the `directShareLink`). Can you make something like this?:
```
const itemsWithDirectShareLink = ['8bb438_1b73a6b067b24175bd087e86613bd00c.jpg', '8bb438_e8735e0102554d6ead385a48e2b9e69a.jpg'].map(createImagesWithDirectShareLink]
```

I think you can reduce the amount of code by 500% that way.
Usually too much mock data makes it difficult to track the difference between the test data